process/project-lifecycle.md

```diff
@@ -0,0 +1,183 @@
+
+<!-- TOC -->
+
+  - [Stages - Definitions & Expectations](#stages---definitions--expectations)
+      - [Sandbox](#sandbox)
+      - [Incubating](#incubating)
+      - [Graduated](#graduated)
+      - [Archived](#archived)
+  - [Submission Process](#submission-process)
+
+<!-- /TOC -->
+
+
+## Stages - Definitions & Expectations
+
+A project's lifecycle stage is a signal of maturity level. Projects progress through the lifecycle by demonstrating their maturity in technical vision, impact to the open source community (such as adoption), and sustainability in governance and contributors. All projects, at all stages, are expected to abide by the [OpenSSF Code of Conduct](https://openssf.org/community/code-of-conduct/).
+
+![Project Stages](/process/project-stages.png)
+
+### Sandbox
+
+The OpenSSF Sandbox is the entry point for early stage Projects and has four goals:
+
+* Encourage experimentation and open collaboration that can add value to the OpenSSF mission and build the ingredients of a successful Incubation level Project.
+* Facilitate alignment with existing Working Groups or Projects if this is desired (there may be cases where pursuit of different approaches is desirable).
+* Nurture Projects.
+* Remove possible legal and governance obstacles to adoption and contribution by ensuring all projects adhere to OpenSSF legal, code of conduct, and IP Policy requirements.
+
+#### Project Responsibilities
+* Provides bi-annual updates to the TAC on technical vision and progress on vision.
+* Maintains a diversified contributor base (i.e. not a single-vendor project).
+* Provides project updates to OpenSSF Marketing Committee as requested.
+
+#### Project Support
+* Receives guidance on technical direction from TAC.
+* Receives consideration as in-scope for any submission to an OpenSSF-managed conference or event.
+* Receives OpenSSF Code of Conduct Committee support.
+* Reserved space for project updates in OpenSSF newsletters.
+* May request infrastructure support from the OpenSSF.
+* Projects, sponsors, and contributors may not promote the project with the OpenSSF logo or be referred to as an "OpenSSF Project" or "OpenSSF $ProjectName." Projects may say they are, "A sandbox project in the OpenSSF" or "An experimental project in the OpenSSF."
+
+#### Sandbox Entry Requirements and Considerations
+
+* Projects must have a minimum of two maintainers with different organization affiliations.
+* Projects must be aligned with the OpenSSF mission _and_ either be a novel approach for existing areas, address an unfulfilled need, or be initial code needed for OpenSSF WG work. It is preferred that extensions of existing OpenSSF projects collaborate with the existing project rather than seek a new project.
```

Comment:
```suggestion
* Projects must be aligned with the OpenSSF mission _and_ either be a novel approach for existing areas or address an unfulfilled need. It is expected that the initial code needed for an OpenSSF WG to work be kept within their repository and will not function as a project in its own right. Should initial WG code grow and mature that it warrants its own Project status, then it is subject to Sandbox entry requirements. It is preferred that extensions of existing OpenSSF projects collaborate with the existing project rather than seek a new project.
```